apps/condo/domains/meter/access/Meter.js

```diff
@@ -29,21 +39,29 @@ async function canReadMeters ({ authentication: { item: user } }) {
         }
     }
 
-    return {
+    const accessFilterForB2BAppServiceUser = await b2bAppServiceUserCanReadObjects(args)
+    return mergeAccessFilters(accessFilterForB2BAppServiceUser, {
```

Comment:
It’s not a good idea to merge. It’s override.

Reply:
removed this